Analysis/include/BaseEventAnalyzer.h

```diff
@@ -65,7 +65,7 @@ class BaseEventAnalyzer : public EventAnalyzerCore {
     std::map<std::string,std::shared_ptr<DYModelBase>> dymod;
     std::shared_ptr<NonResModel> nonResModel;
     const std::vector<std::string> trigger_patterns;
-    // const std::vector<std::string> trigger_patterns_vbf;
+    bool pass_vbf_trigger;
```

Comment:
don't create it as a class member, because it is event-dependent. Pass as an argument to the`DetermineEventCategories` instead.